examples/reference/panes/VTK.ipynb

```diff
@@ -21,7 +21,6 @@
     "For layout and styling related parameters see the [customization user guide](../../user_guide/Customization.ipynb).\n",
     "\n",
     "* **``camera``** (dict): A dictionary reflecting the current state of the VTK camera\n",
-    "* **``object``** (dict or altair Chart): Either a dictionary containing a vega or vega-lite plot specification or an altair Chart\n",
```

Comment:
This was of course a stupid copy/paste error but object should definitely still be mentioned here.